pandas/tests/io/excel/test_readers.py

```diff
@@ -897,12 +897,6 @@ def test_read_excel_bool_header_arg(self, read_ext):
             with pytest.raises(TypeError, match=msg):
                 pd.read_excel("test1" + read_ext, header=arg)
 
-    def test_read_excel_chunksize(self, read_ext):
```

Comment:
does this work now?

Reply:
I've removed ``**kwds`` from the signature, so passing ``chunksize`` into ``read_excel`` isn't even possible any more and testing if it fails makes no sense now.

More generally, previously any keyword argument could be passed, so e.g. ``read_excel("file.xlsx", blabla=1)`` didn't fail, but shoild fail.

Reply:
ok can you add a test then for extraneous keywords?

Reply:
IMO that would be too preemptive. Someone could of course later add a ``**kwargs`` but they could also do so much else wrong...